static/app/views/issueDetails/traceTimeline/traceTimeline.tsx

```diff
@@ -49,28 +64,49 @@ export function TraceTimeline({event}: TraceTimelineProps) {
 
   return (
     <ErrorBoundary mini>
-      <TimelineWrapper>
-        <div ref={timelineRef}>
-          <TimelineEventsContainer>
-            <TimelineOutline />
-            {/* Sets a min width of 200 for testing */}
-            <TraceTimelineEvents event={event} width={Math.max(width, 200)} />
-          </TimelineEventsContainer>
-        </div>
-        <QuestionTooltipWrapper>
-          <QuestionTooltip
-            size="sm"
-            title={t(
-              'This is a trace timeline showing all related events happening upstream and downstream of this event'
-            )}
-            position="bottom"
-          />
-        </QuestionTooltipWrapper>
-      </TimelineWrapper>
+      {timelineStatus === 'shown' ? (
+        <TimelineWrapper>
+          <div ref={timelineRef}>
+            <TimelineEventsContainer>
+              <TimelineOutline />
+              {/* Sets a min width of 200 for testing */}
+              <TraceTimelineEvents event={event} width={Math.max(width, 200)} />
+            </TimelineEventsContainer>
+          </div>
+          <QuestionTooltipWrapper>
+            <QuestionTooltip
+              size="sm"
+              title={t(
+                'This is a trace timeline showing all related events happening upstream and downstream of this event'
+              )}
+              position="bottom"
+            />
+          </QuestionTooltipWrapper>
+        </TimelineWrapper>
+      ) : (
+        <Feature features="related-issues-issue-details-page">
+          {timelineSkipped && (
+            <div style={{width: '400px'}}>
```

Comment:
This should be a styled component (although I know this is temporary so probably fine for now)

Reply:
I will handle this in the next PR.